coffee_machine_project/main.py

```diff
@@ -0,0 +1,86 @@
+MENU = {
+    "espresso": {
+        "ingredients": {
+            "water": 50,
+            "coffee": 18,
+        },
+        "cost": 1.5,
+    },
+    "latte": {
+        "ingredients": {
+            "water": 200,
+            "milk": 150,
+            "coffee": 24,
+        },
+        "cost": 2.5,
+    },
+    "cappuccino": {
+        "ingredients": {
+            "water": 250,
+            "milk": 100,
+            "coffee": 24,
+        },
+        "cost": 3.0,
+    }
+}
+
+profit = 0
+resources = {
+    "water": 300,
+    "milk": 200,
+    "coffee": 100,
+}
+
+def is_resources_sufficient(order_ingredients):
+  for item in order_ingredients:
+    if order_ingredients[item] >= resources[item]:
+        print(f"Sorry there is not enough water{item}.")
+        return False
+  return True
+
+
+def process_coins():
+  print("Please insert coins.")
+  total = int(input("How many quarters?: ")) *0.25
+  total += int(input("How many dimes?: ")) *0.01
+  total += int(input("How many nickles?: ")) *0.05
+  total += int(input("How many pennies?: ")) *0.01
+  return total
+
+
+def is_transaction_successful(money_received, drink_cost):
+  if money_received >= drink_cost:
+    change = round(money_received - drink_cost, 2)
+    print(f"Here is ${change} in change.")
+    global profit
+    profit += drink_cost
+    return True
+  else:
+    print("Sorry that's not enough money. Money refunded.")
+    return False
+
+
+def make_coffee(drink_name, order_ingredients):
+  for items in order_ingredients:
+    resources[items] -= order_ingredients[items]
+  print(f"Here is your {drink_name} ")
+
+
+is_on = True
+
+
+while is_on:
+  choice = input("What would you like? (espresso/latte/cappuccino):")
+  if choice == "off":
+    is_on = False
+  elif choice == "report":
+    print(f"Water: {resources['water']}ml")
+    print(f"Milk: {resources['milk']}ml")
+    print(f"Coffee: {resources['milk']}g")
+    print(f"Money: ${profit}")
+  else:
+    drink = MENU[choice]
+    if is_resources_sufficient(drink["ingredients"]):
+      payment = process_coins()
+      if is_transaction_successful(payment, drink["cost"]):
+        make_coffee(choice, drink["ingredients"])
```

Comment:
Going forward, can you switch to a 4-space indent for your code? Let me know if you don't know what I mean or don't know how to configure that in VSCode.

Reply:
Do you mean "Tab key"?

Reply:
Ah sorry, because that time I was doing it on Replit then I transfer it on VSCode to pushed it on my repo. I should have check it first.